From f5fefa32f1ac59d2a9c00221281fe3541845d6d2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?G=C3=B6kt=C3=BCrk=20Y=C3=BCksek?= <gokturk@gentoo.org>
Date: Mon, 16 Jul 2018 18:21:21 -0400
Subject: [PATCH] doc: look for gobject_typesfile in build dir

Looking for playerctl.types file in the source directory is a mistake
because it's generated during build and placed in build dir. Look for
the types file in build dir instead.
---
 doc/reference/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/reference/meson.build b/doc/reference/meson.build
index 100d561..b5a16a7 100644
--- a/doc/reference/meson.build
+++ b/doc/reference/meson.build
@@ -19,7 +19,7 @@ gnome.gtkdoc(
     '--name-space=meson.project_name()',
   ],
   scan_args: '--deprecated-guards="PLAYERCTL_DISABLE_DEPRECATED"',
-  gobject_typesfile: meson.project_name() + '.types',
+  gobject_typesfile: join_paths(meson.current_build_dir(), meson.project_name() + '.types'),
   fixxref_args: [
     '--extra-dir=@0@'.format(join_paths(glib_docpath, 'gobject')),
     '--extra-dir=@0@'.format(join_paths(glib_docpath, 'gio')),
-- 
2.17.0

